GithubApp/Extensions/ImageViewExtension.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  ImageViewExtension.swift
+//  GithubApp
+//
+//  Created by BrainX on 08/06/2022.
+//
+
+import UIKit
+extension UIImageView {
+    
+    // MARK: - Function To Get Image from URL
```

Comment:
pragma mark does not need to be this long. Also function name should clarify what its doing. pragma mark should be a category or a type. like public methods, private method, actions etc.